nuxeo-features/nuxeo-platform-rendition/nuxeo-platform-rendition-core/src/test/java/org/nuxeo/ecm/platform/rendition/service/TestDefaultRendition.java

```diff
@@ -0,0 +1,104 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Guillaume Renard <grenard@nuxeo.com>
+ */
+package org.nuxeo.ecm.platform.rendition.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+import javax.inject.Inject;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.nuxeo.ecm.collections.api.CollectionManager;
+import org.nuxeo.ecm.core.api.Blob;
+import org.nuxeo.ecm.core.api.CloseableFile;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.platform.rendition.Renderable;
+import org.nuxeo.ecm.platform.rendition.Rendition;
+import org.nuxeo.runtime.api.Framework;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+
+/**
+ * @since 9.3
+ */
+@RunWith(FeaturesRunner.class)
+@Features(RenditionFeature.class)
+@Deploy({ "org.nuxeo.ecm.platform.collections.core" })
+public class TestDefaultRendition {
+
+    @Inject
+    protected CoreSession session;
+
+    @Inject
+    protected RenditionService renditionService;
+
+    /**
+     * @since 9.3
+     */
+    @Test
+    public void testDefaultRenditionOnCollection() throws Exception {
+        DocumentModel folder1 = session.createDocumentModel("/", "dummy-folder", "Folder");
+        folder1 = session.createDocument(folder1);
+        DocumentModel file01 = TestRenditionProvider.createBlobDoc(folder1.getPathAsString(), "File", session);
+        DocumentModel file02 = TestRenditionProvider.createBlobDoc(folder1.getPathAsString(), "File", session);
+        DocumentModel collection = session.createDocumentModel(folder1.getPathAsString(), "dummy-collection",
+                "Collection");
+        collection = session.createDocument(collection);
+        DocumentModel file11 = TestRenditionProvider.createBlobDoc("File", session);
+        DocumentModel file12 = TestRenditionProvider.createBlobDoc("File", session);
+        CollectionManager collectionManager = Framework.getService(CollectionManager.class);
+        collectionManager.addToCollection(collection, file11, session);
+        collectionManager.addToCollection(collection, file12, session);
+
+        Rendition ren = renditionService.getDefaultRendition(folder1, "download", null);
+        assertNotNull(ren);
+        Blob resultBlob = ren.getBlob();
+        assertEquals("application/zip", resultBlob.getMimeType());
+        try (CloseableFile source = resultBlob.getCloseableFile()) {
+            try (ZipFile zip = new ZipFile(source.getFile())) {
+                assertEquals(3, zip.size());
+                ZipEntry z1 = zip.getEntry("dummy-collection");
+                assertNotNull(z1);
+                String name = z1.getName();
```

Comment:
Should you assert something on the name? And maybe check if the collection zip entry contains its 2 member renditions?